README.rst

```diff
@@ -216,31 +218,46 @@ IP addresses usage:
 .. code:: yaml
 
   zookeeper:
-    nodes:
-      - 192.168.0.101
-      - 192.168.0.102
-      - 192.168.0.103
-
+    clusters:
+      - nodes:
+        - 192.168.0.101
+        - 192.168.0.102
+        - 192.168.0.103
+      - nodes:
+        - 192.168.1.101
+        - 192.168.1.102
+        - 192.168.1.103
```

Comment:
As for me, this data structure looks very... pardon, creepy :smiley: 
Instead of having *list of dicts* with *list of values*, why not to convert that into the simple *list of lists*?
For example:
```yaml
zookeeper:
  clusters:
    - - 192.168.0.101
      - 192.168.0.102
      - 192.168.0.103
    - - 192.168.1.101
      - 192.168.1.102
      - 192.168.1.103
```
This would make implementation much simpler. For now, having the same multiple `nodes` keys doesn't make any sense.

Or even better, to be able to merge the node list from different PIllar SLS using includes or via the top file, maybe it would be better to have the "cluster name" as a dictionary key? Like this:
```yaml
zookeeper:
  clusters:
    cluster1:
      - 192.168.0.101
      - 192.168.0.102
      - 192.168.0.103
    cluster2:
      - 192.168.1.101
      - 192.168.1.102
      - 192.168.1.103
```
That's how you can distinguish the clusters and put the name somewhere for debugging purposes. This is just a suggestion, but will be good to have.